terraform/configure.py

```diff
@@ -50,7 +50,9 @@
         "mgmt_nessus": 0,
     },
     "felddy": {"nmap": 4, "nessus": 1, "mongo": 1, "mgmt_bastion": 0, "mgmt_nessus": 0},
+    "hillary": {"nmap": 0, "nessus": 0, "mongo": 1, "mgmt_bastion": 0, "mgmt_nessus": 0},
```

Comment:
You might as well add a `daver` workspace while you are adding the others.

Reply:
See commit 3bc437c.